src/main.rs

```diff
@@ -48,6 +50,20 @@ fn main() {
     });
     info!("Data plane started");
 
+    thread::sleep(Duration::from_millis(100));
+
+    let mut pubsub = PubSub::default();
```

Comment:
This looks like test case related stuff, did you mean to leave this in main?

Reply:
thx - removed